domain.yml

```diff
@@ -15,63 +37,95 @@ entities:
 
 responses:
   utter_greet:
-  - text: "Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer a few questions about your health?"
+  - text: Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer few questions about your health?
+
+  utter_cheer_up:
```

Comment:
I see that the moodbot dialogues have been added to the wellness check bot. What was the reason for including these?

Reply:
Yes you are right. I am removing all mood bot utterances. I started from `rasa init` & must have forgotten to remove default things.